[2023-09-04T06:01:53.878+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.delete_data_from_table scheduled__2022-02-19T00:00:00+00:00 [queued]>
[2023-09-04T06:01:53.897+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.delete_data_from_table scheduled__2022-02-19T00:00:00+00:00 [queued]>
[2023-09-04T06:01:53.898+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2023-09-04T06:01:54.099+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): delete_data_from_table> on 2022-02-19 00:00:00+00:00
[2023-09-04T06:01:54.110+0000] {standard_task_runner.py:57} INFO - Started process 5065 to run task
[2023-09-04T06:01:54.115+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v04', 'delete_data_from_table', 'scheduled__2022-02-19T00:00:00+00:00', '--job-id', '317', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmpat2qiuyo']
[2023-09-04T06:01:54.119+0000] {standard_task_runner.py:85} INFO - Job 317: Subtask delete_data_from_table
[2023-09-04T06:01:54.204+0000] {task_command.py:415} INFO - Running <TaskInstance: dag_with_postgres_operator_v04.delete_data_from_table scheduled__2022-02-19T00:00:00+00:00 [running]> on host 10bdc2e98230
[2023-09-04T06:01:54.345+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v04' AIRFLOW_CTX_TASK_ID='delete_data_from_table' AIRFLOW_CTX_EXECUTION_DATE='2022-02-19T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-02-19T00:00:00+00:00'
[2023-09-04T06:01:54.347+0000] {sql.py:274} INFO - Executing: 
            delete from dag_runs where dt = '2022-02-19' and dag_id = 'dag_with_postgres_operator_v04';
        
[2023-09-04T06:01:54.360+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:01:54.370+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:01:54.386+0000] {sql.py:418} INFO - Running statement: 
            delete from dag_runs where dt = '2022-02-19' and dag_id = 'dag_with_postgres_operator_v04';
        , parameters: None
[2023-09-04T06:01:54.391+0000] {sql.py:427} INFO - Rows affected: 0
[2023-09-04T06:01:54.408+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v04, task_id=delete_data_from_table, execution_date=20220219T000000, start_date=20230904T060153, end_date=20230904T060154
[2023-09-04T06:01:54.448+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-04T06:01:54.483+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-09-04T06:04:37.927+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.delete_data_from_table scheduled__2022-02-19T00:00:00+00:00 [queued]>
[2023-09-04T06:04:37.950+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.delete_data_from_table scheduled__2022-02-19T00:00:00+00:00 [queued]>
[2023-09-04T06:04:37.951+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2023-09-04T06:04:37.976+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): delete_data_from_table> on 2022-02-19 00:00:00+00:00
[2023-09-04T06:04:37.985+0000] {standard_task_runner.py:57} INFO - Started process 6328 to run task
[2023-09-04T06:04:37.988+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v04', 'delete_data_from_table', 'scheduled__2022-02-19T00:00:00+00:00', '--job-id', '858', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmpdf9sscb5']
[2023-09-04T06:04:37.991+0000] {standard_task_runner.py:85} INFO - Job 858: Subtask delete_data_from_table
[2023-09-04T06:04:38.074+0000] {task_command.py:415} INFO - Running <TaskInstance: dag_with_postgres_operator_v04.delete_data_from_table scheduled__2022-02-19T00:00:00+00:00 [running]> on host 10bdc2e98230
[2023-09-04T06:04:38.226+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='mohghaff' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v04' AIRFLOW_CTX_TASK_ID='delete_data_from_table' AIRFLOW_CTX_EXECUTION_DATE='2022-02-19T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-02-19T00:00:00+00:00'
[2023-09-04T06:04:38.229+0000] {sql.py:274} INFO - Executing: 
            delete from dag_runs where dt = '2022-02-19' and dag_id = 'dag_with_postgres_operator_v04';
        
[2023-09-04T06:04:38.247+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:04:38.260+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:04:38.277+0000] {sql.py:418} INFO - Running statement: 
            delete from dag_runs where dt = '2022-02-19' and dag_id = 'dag_with_postgres_operator_v04';
        , parameters: None
[2023-09-04T06:04:38.284+0000] {sql.py:427} INFO - Rows affected: 1
[2023-09-04T06:04:38.303+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v04, task_id=delete_data_from_table, execution_date=20220219T000000, start_date=20230904T060437, end_date=20230904T060438
[2023-09-04T06:04:38.361+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-04T06:04:38.396+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
